### ISSUE TITLE
New release with repaired makefile

### ISSUE THREAD

**Issue by zen0bit** [BashGui/easybashgui#26]
Looks like makefile in last release doesn't work. Already repaired? (I think) New release?

```
Copying easybashgui documentation ...
mkdir -p /destdir//easybashgui-12.0.5/usr/share/doc/easybashgui
cp EasyBashGUI-license README /destdir//easybashgui-12.0.5/usr/share/doc/easybashgui
cp: cannot stat 'README': No such file or directory
make: *** [makefile:55: install] Error 1
=> ERROR: easybashgui-12.0.5_1: do_install: '${make_cmd} STRIP=true PREFIX=/usr prefix=/usr DESTDIR=${DESTDIR} ${make_install_args} ${make_install_target}' exited with 2
=> ERROR:   in do_install() at common/build-style/gnu-makefile.sh:42
```

**Comment by mckaygerhard** [comment]
hi, @zen0bit it seems was fixed with #25  .. as i know with commit c346c342edf24c39bd1e105630cd905ba3b95947 please  check again..

**Comment by zen0bit** [comment]
> hi, @zen0bit it seems was fixed with #25 .. as i know with commit [c346c34](https://github.com/BashGui/easybashgui/commit/c346c342edf24c39bd1e105630cd905ba3b95947) please check again..

yes (working for me), but merged after release..

So current latest don't work.

It's why I asked for new release.

Wanna push it to void's repos..

**Comment by vaisarger** [comment]
Good man @mckaygerhard fixed this bug...

Thank you very much!
